integration_tests/mockApis/prisonApi.ts

```diff
@@ -0,0 +1,76 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+export default {
+  stubPing: (): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPattern: '/prisonApi/health/ping',
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: { status: 'UP' },
+      },
+    })
+  },
+  stubGetUserCaseLoads: (): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPattern: '/prisonApi/api/users/me/caseLoads',
+      },
+      response: {
+        status: 200,
+        headers: {
+          'Content-Type': 'application/json;charset=UTF-8',
+        },
+        jsonBody: [
+          {
+            caseLoadId: 'MDI',
+            description: 'Moorland (HMP & YOI)',
+            currentlyActive: true,
+            type: 'INST',
+          },
+        ],
+      },
+    })
+  },
+  stubGetAgencyLocations: (): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPattern: '/prisonApi/api/agencies/MDI/locations',
```

Comment:
Wrong Endpoint:  Should be `{{prison_api}}/api/users/me/locations`

Reply:
wouldn't this other `/api/users/me/locations` endpoint return all locations accessible to the user tho? Including ones for a case load which is not the active one?